tests/python/python_binding_tests.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python3
```

Comment:
This is the change that broke the tests. Please make sure your editor doesn't accidentally delete the shebang line!